frontend/src/metabase/app.js

```diff
@@ -66,6 +66,16 @@ const theme = {
   space: [4, 8, 16, 32, 64, 128],
 };
 
+console.log(MetabaseSettings.get("application-font"));
+
+const applicationFontStyles = css`
```

Comment:
WDYT about moving these styles to somewhere else, e.g. to `styled-components` or even `styled-components/theme` folder?